pactman/verifier/broker_pact.py

```diff
@@ -39,7 +40,10 @@ def get_broker_navigator(self):
 
     def consumers(self):
         nav = self.get_broker_navigator()
-        broker_provider = nav['latest-provider-pacts'](provider=self.provider_name)
+        try:
+            broker_provider = nav['latest-provider-pacts'](provider=self.provider_name)
+        except Exception as e:
+            raise ValueError(f'error fetching pacts from {self.pact_broker_url} for {self.provider_name}: {e.exception}')
```

Comment:
It feels risky to reference `e.exception` here since we don't know this attribute will exist on the exception. It could lead to some confusing error messages. Perhaps do something like:

```
if hasattr(e, 'exception'):
    raise ValueError(f'error fetching pacts from {self.pact_broker_url} for {self.provider_name}: {e.exception}')
else:
    raise
```

Reply:
I was thinking more on the lines of 

`raise ValueError('..') from e` ?

Reply:
I've fixed the access of `e.exception` to just format `e` into the message.